### PR TITLE
Add repository section

### PR DESCRIPTION
**Pull request by diegolovison**

### CODE REVIEW

File: wg-pipelines/README.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ The [charter](charter.md) defines the scope and governance of the Pipelines Work
 - GitHub Teams:
     - [@kubeflow/wg-pipelines-bugs](https://github.com/orgs/kubeflow/teams/wg-pipelines-bugs) - Bug Triage and Troubleshooting
     - [@kubeflow/wg-pipelines-reviews](https://github.com/orgs/kubeflow/teams/wg-pipelines-reviews) - API Changes and Reviews (API Machinery APIs, NOT all APIs)
+
+## Repository
+- https://github.com/kubeflow/pipelines
+- https://github.com/kubeflow/kfp-tekton
```
[the remark this answers is not in the document]
Comment (terrytangyuan) [reply wg-pipelines/README.md:33]: cc @Tomcli

[the remark this answers is not in the document]
Comment (rimolive) [reply wg-pipelines/README.md:33]: True that, but still needs some pointers as there are pending issues from that side. We need to figure out how to handle this.

But I agree that it should be at least marked as deprecated. @diegolovison Can you do this?

**Comment by thesuperzapper** [comment]
@diegolovison This is an automatically generated file, please do NOT update it directly.

/hold

**Comment by rimolive** [comment]
@thesuperzapper Where this should be added? What we want to do with this change is tracking the component repos that belongs to pipelines WG.

**Comment by diegolovison** [comment]
Closed in favor of https://github.com/kubeflow/community/blob/master/wgs.yaml

**Pull request closed without merging**